src/engine/settingsengine.cpp

```diff
@@ -83,7 +83,7 @@ void SettingsEngine::initialize() {
         std::string sceneFile = _scenes.getDescriptionByValue(_scenes);
         OsEng.configurationManager().setValue(
             ConfigurationManager::KeyConfigScene, sceneFile);
-        OsEng.renderEngine().scene()->scheduleLoadSceneFile(sceneFile);
+        OsEng.scheduleLoadScene("${SCENE}/" + sceneFile);
```

Comment:
I agree, but dragons were here before as well. I'll clarify the dragons by passing in the constant sceneDir instead. Since the scene files are collected in the same method, I think this is fine until future refactors.